.circleci/config.yml

```diff
@@ -352,6 +358,43 @@ jobs:
             RELEASE_CHANNEL: experimental
           command: yarn test-build-prod --maxWorkers=2
 
+  test_ms_windows:
+    executor:
+      name: win/default
+    environment: *ms_windows_environment
+    steps:
+      - run:
+          # Fix line endings in Windows.
+          command: git config --global core.autocrlf input
+      - checkout
+      - restore_cache:
+          keys:
+            - v2-win-node-{{ arch }}-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - v2-win-node-{{ arch }}-{{ .Branch }}-
+            - v2-win-node-{{ arch }}-
```

Comment:
We are using a different cache key for Win and Linux, since the files and folders might not be exactly the same - so we can't use the same restore step.